.github/ISSUE_TEMPLATE/rfe.yaml

```diff
@@ -0,0 +1,25 @@
+name: Feature Request
+description: Request for Feature Enhancement
+title: "[RFE]: "
+labels: ["feature", "triage"]
+assignees: @openshift/openshift-team-windows-containers 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This form is for requesting a change in expected behavior for WMCO.
+  - type: input
+    id: problem-statement
+    attributes:
+      label: Problem Statement
+      description: A short description of the core problem that needs to be solved
+      placeholder: I want to be able to use private container registries 
```

Comment:
Does this text need to be a double quoted string? I'm not sure a placeholder is needed here anyway

Reply:
> Does this text need to be a double quoted string?

No it doesn't

I like the placeholder as it guides the issue writer to use a user story approach when describing the problem they have